lib/hammer_cli_foreman_rh_cloud/ping.rb

```diff
@@ -0,0 +1,33 @@
+require 'hammer_cli_foreman/ping'
+
+module HammerCLIForemanRhCloud
+  class PingCommand < HammerCLIForeman::Command
+    resource :ping
+
+    output do
+      from 'foreman_rh_cloud' do
+        from :services do
+          label 'advisor' do
+            from 'advisor' do
+              field 'status', _('Status')
+              field '_response', _('Server Response')
+            end
+          end
+          label 'vulnerability' do
+            from 'vulnerability' do
+              field 'status', _('Status')
+              field '_response', _('Server Response')
+            end
+          end
+        end
+      end
+    end
+  end
+
+  HammerCLIForeman::PingCommand.subcommand 'foreman-rh-cloud',
+    HammerCLIForemanRhCloud::PingCommand.desc,
+    HammerCLIForemanRhCloud::PingCommand
+  HammerCLIForeman::PingCommand::ForemanCommand.extend_with(
```

Comment:
I see the commands are self-registering. If it's the case, I would expect `command_extensions/ping` file to self-register. Otherwise you have created a dependency between files without a proper require.